website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -1,10 +1,11 @@
 ---
 layout: docs
-page_title: Connect - Envoy Integration
-description: Consul Connect has first-class support for configuring Envoy proxy.
+page_title: Envoy Proxy Configuration | Service Mesh
+description: >-
+Consul supports Envoy proxies to direct traffic throughout the service mesh. Learn about Consul versions and their Envoy support, and use the reference guide to review options for bootstrap configuration, dynamic configuration, and advanced topics like escape hatch overrides.
```

Comment:
This makes it sound like Envoy is _an_ option, when it's really the best, most supported option. Envoy should be used by Consul service mesh users except in rare circumstances. Should we make that clearer in the description? Maybe that's something that just belongs on the page itself?